provider/oci/instance.go

```diff
@@ -177,7 +179,7 @@ func (o *ociInstance) waitForPublicIP(ctx envcontext.ProviderCallContext) error
 			return err
 		}
 		if iteration >= 30 {
-			logger.Warningf("Instance still in running state after %v checks. breaking loop", iteration)
+			logger.Warningf("Still no public IP after %v checks. breaking loop", iteration)
```

Comment:
I'm happy with the change, I'm not sure that "breaking loop" should be in a warning message.